app/database.py

```diff
@@ -3,8 +3,9 @@
 from sqlalchemy.orm import sessionmaker
 
 # Database configurations
-SQLALCHEMY_DATABASE_URL = "sqlite:///./test.db"
-engine = create_engine(SQLALCHEMY_DATABASE_URL, connect_args={"check_same_thread": False})
+SQLALCHEMY_DATABASE_URL = "postgresql://sage:sage@localhost:5432/sage"
```

Comment:
I think you need to change the credentials here. Use something proper for your project